tests/main.test.ts

```diff
@@ -31,7 +34,12 @@ describe("Wallet command tests", () => {
   });
 
   it("Should link a wallet", async () => {
-    const spy = jest.spyOn(console, "log");
+    jest
+      .spyOn(RPCHandler.prototype, "getFirstAvailableRpcProvider")
+      .mockResolvedValue(new JsonRpcProvider({ url: "https://fastethrpc.com", skipFetchSetup: true }, Number(1)));
```

Comment:
I think it would make more sense to just use `passthrough` from `mwjs` and actually use our real RPC package if possible.

Reply:
Sometimes the test failed because it wasnt able to resolve an ENS. 
https://github.com/ubiquity-os-marketplace/command-wallet/pull/19#issuecomment-2428543283

Reply:
Yes this happens. I guess it's fine for tests, but ideally our RPC shouldn't fail.

Reply:
@0x4007 What do you think would be the best approach? In the context of testing, hardcoding an RPC that we are sure works seems ok, but having our RPCs would be preferable but more unstable.

Reply:
I'm not sure if its valid to skip implementing this test temporarily until we fix our RPC provider. I plan to work on it on the plane ride all day tomorrow so hopefully I have something to show for it. I might be able to push support for ENS only first and then work on generalizing the encoding/decoding of any contract method.

Reply:
Shall I undo this change then?

Reply:
I was notified that some fixes were made to either the RPC manager or one of the interfaces that implements it, but I would always think that an RBC manager would be more stable than a single RPC end point. 

RPC endpoints seem to not have hundred percent up time